vpr/src/route/rr_node.h

```diff
@@ -185,6 +185,7 @@ struct t_rr_indexed_data {
     float T_linear = std::numeric_limits<float>::quiet_NaN();
     float T_quadratic = std::numeric_limits<float>::quiet_NaN();
     float C_load = std::numeric_limits<float>::quiet_NaN();
+    int number_of_nodes = 0;
```

Comment:
Does this value need to be kept around?  If not, remove it from this structure and keep it temporarily.

Reply:
No, it was a leftover. Removed